frontend/src/metabase/services.js

```diff
@@ -480,7 +480,9 @@ export function setEmbedQuestionEndpoints(token) {
 
 export function setEmbedDashboardEndpoints() {
   if (!IS_EMBED_PREVIEW) {
-    setDashboardEndpoints("/api/embed");
+    setDashboardEndpoints(embedBase);
+  } else {
+    setDashboardParameterValuesEndpoint(embedBase);
   }
```

Comment:
nit: double-negative if-else, maybe invert this?

Reply:
I was contemplating that, but we have the block above which uses the same condition.

https://github.com/metabase/metabase/blob/f671c2d6de773d58f10dcb902a48ff1559e878f3/frontend/src/metabase/services.js#L476-L478

it would be weird to revert the condition on this block to early return imo. That's why I keep it like this so they are both easier to understand.

Does this make sense?

Reply:
Ah, if it's for consistency then I'm fine with that 👍🏻